cmd/create/decision/cmd.go

```diff
@@ -0,0 +1,107 @@
+package decision
+
+import (
+	"context"
+	"strings"
+
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+
+	v1 "github.com/openshift-online/ocm-sdk-go/accesstransparency/v1"
+	"github.com/spf13/cobra"
+	errors "github.com/zgalor/weberr"
+
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	use     = "decision"
+	short   = "Create a decision for an access request"
+	long    = short
+	example = `  # Create a decision for an access request to approve it
+  rosa create decision --access-request <access_request_id> --decision Approved
+  `
+)
+
+type DecisionOptions struct {
```

Comment:
NP - can we name this `Options` instead since the package name is "decision"? This avoids stuttering when using the type in code, i.e. `decision.DecisionOptions` vs. `decision.Options`.

This is also called out in "Naming package contents" here: https://go.dev/blog/package-names

Reply:
+1
updated the name, thanks for the suggestion